master/buildbot/newsfragments/force-password-into-bytes.bugfix

```diff
@@ -0,0 +1 @@
+Updated ``UserPasswordAuth`` to force user passwords into bytes
```

Comment:
please be more user oriented for relnotes.
Fix incompatibility issue of UserPasswordAuth with python 3.